### PR TITLE
Added support for Ubuntu

### PR DESCRIPTION
**Pull request by prahastakumar**
Added support for Ubuntu

### CODE REVIEW

**Comment by eyalneumann** [comment]
wasn't this feature discontinued?

**Review by OmprakashPaliwal — APPROVED** [review]

**Comment by OmprakashPaliwal** [comment]
The two comments are cosmetics. merging as is

File: tasks/redhat-prerequisite.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# required dependencies
+
+- name: make sure prerequisites are installed
+  yum:
+    name: "{{ item }}"
+    state: present
+    lock_timeout: 180
+  become: yes
+  with_items:
+    - epel-release
```
[the remark this answers is not in the document]
Comment (prahastakumar) [reply tasks/redhat-prerequisite.yml:10]: I think epel was installed to get some extra packages

File: tasks/debian-prerequisite.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+- name: make sure prerequisites are installed
+  apt:
+    name: "{{ item }}"
+    state: present
+    lock_timeout: 180
+  become: yes
+  with_items:
+    - rsync
```
[the remark this answers is not in the document]
Comment (prahastakumar) [reply tasks/debian-prerequisite.yml:8]: correct don't see any usage of rsync in role

**Pull request merged**